vsts_ci/azure-pipelines.yml

```diff
@@ -140,3 +140,60 @@ jobs:
         inputs:
           pathtoPublish: $(Build.ArtifactStagingDirectory)
           artifactName: build-artifact-drop
+
+      - task: Bash@3
+        inputs:
+          targetType: 'inline'
+          script: |
+            if [[ $(Build.SourceBranch) =~ ^refs/tags/v?[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "##vso[task.setvariable variable=PYPI_PUBLISH_FLAG;isOutput=true]true"
+            fi
+        name: PublishFlag
+
+  - job: UploadToStorage
+    dependsOn: 
+      - MacOS
+      - Windows
+      - Linux
+      - PublishDropFile
+      - PublishStandaloneBinariesForWin32
+    condition: and(succeeded(), eq(dependencies.PublishDropFile.outputs['PublishFlag.PYPI_PUBLISH_FLAG'], 'true'))
+    pool:
+      name: Azure-IoT-EdgeExperience-1ES-Hosted-Windows
+      demands:
+        - ImageOverride -equals MMS2022TLS
+    
+    steps:
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        buildType: 'current'
+        artifactName: 'build-artifact-drop'
+        targetPath: '$(Build.ArtifactStagingDirectory)/Package'
+
+    - task: PowerShell@2
+      inputs:
+        targetType: 'inline'
+        script: |
+          $VersionFormat = Get-Content .\iotedgehubdev\__init__.py | select-string -pattern "version"
+          $PackageVersion = (($VersionFormat -split "=")[1] -split "'")[1]
+          write-host "##vso[task.setvariable variable=version;]$PackageVersion"
+
+    - task: PowerShell@2
+      inputs:
+        targetType: 'inline'
+        script: write-host $(version)
```

Comment:
can these 2 tasks be merged in one?

Reply:
I wrote that to test if it it is taking correct version we no longer need that